documentation/long-stay-baseline.mdx

```diff
@@ -0,0 +1,528 @@
+
+We also know that length of stay varies by group, so further plots of the ratio of MAE to length of stay are generated in the notebooks, but not included here for brevity.
+
+The final model generated did not adequately capture length of stay across the population. Some sample sizes of demographic groups were too small to draw conclusions, but the process of exploring the underlying distribution of the target feature (length of stay), count (n) and model performance were important and should remain part of future work.
+
+### 6.3 Classification models<a name='classificationmodels'></a>
+
+In addition to predicting the length of stay in days, we are also interested in stratifying the risk of a patient becoming a long stayer. This can be inferred from their predicted length of stay (see [model comparison](#modelcomparison)), but we can also train a classification model to do this directly.
+
+The agreed stratification of risk of long stay is defined as:
+
+Risk Category|Day Range for Risk Category
+-----|------
+1 - Very low risk|0-6
+2 - Low risk|7-10
+3 - Normal risk|11-13
+4 - Elevated risk|14-15
+5 - High risk|>15
+
+We keep the training features the same as in the regression models, and encode risk from the actual length of stay as the target feature.
+
+The classification equivalents of the regression models were selected:
+
+Model|Regression version|Classification version
+---|---|---
+Dummy|[Mean](https://scikit-learn.org/stable/modules/generated/sklearn.dummy.DummyRegressor.html)|[Prior](https://scikit-learn.org/stable/modules/generated/sklearn.dummy.DummyClassifier.html)
+ElasticNet|[ElasticNet](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.ElasticNet.html)|[LogisticRegression](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html)
+Decision Tree|[DecisionTreeRegressor](https://scikit-learn.org/stable/modules/generated/sklearn.tree.DecisionTreeRegressor.html)|[DecisionTreeClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.tree.DecisionTreeClassifier.html)
+Random Forest|[RandomForestRegressor](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestRegressor.html)|[RandomForestClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html)
+XGBoost|[XGBRegressor](https://xgboost.readthedocs.io/en/stable/python/python_api.html#xgboost.XGBRegressor)|[XGBClassifier](https://xgboost.readthedocs.io/en/stable/python/python_api.html#xgboost.XGBClassifier)
+CatBoost|[CatBoostRegressor](https://catboost.ai/en/docs/concepts/python-reference_catboostregressor)|[CatBoostClassifier](https://catboost.ai/en/docs/concepts/python-reference_catboostclassifier)
+
+The training, validation and test regime was the same as for the regression models.
+
+**Class imbalance**
+
+In the training set, we observe the following class imbalance:
```

Comment:
May be can also include the values for the validation set as well?

Reply:
Because the class weights are determined using the training data (so as not to leak information from the validation/test set) I think this is sufficient.

Added a sentence to clarify